source/MRVoxels/MRObjectVoxels.cpp

```diff
@@ -564,6 +564,11 @@ void ObjectVoxels::setSerializeFormat( const char * newFormat )
     serializeFormat_ = newFormat;
 }
 
+void ObjectVoxels::resetFrontColor()
+{
+    setDefaultColors_();
+}
```

Comment:
please add comment in all such places, that we cannot do 
```
setDefaultColors_
{
resetColors();
}
```
to keep `setDefaultColors_` non-virtual and `resetColors` - virtual